front/app/components/UI/FileUploader/FileInput.tsx

```diff
@@ -133,6 +133,17 @@ const fileAccept = [
   'video/x-matroska',
 
   '.jpg',
+  'image/jpg',
+  '.jpeg',
+  'image/jpeg',
+  '.gif',
+  'image/gif',
+  '.png',
+  'image/png',
+  '.webp',
+  'image/webp',
+  '.svg',
+  'image/svg+xml',
```

Comment:
Weirdly, this wasn't needed for the BE additions to the `PhaseFileUploader` to work (via the FE), but was need for the `ProjectFileUploader` BE additions to work via the FE.

Reply:
Cannot comment on this 😄